plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java

```diff
@@ -223,7 +226,7 @@ public void onEnable() {
 
         this.guildValidationTask = Bukkit.getScheduler().runTaskTimerAsynchronously(this, new GuildValidationHandler(this), 100L, 20L);
         this.tablistBroadcastTask = Bukkit.getScheduler().runTaskTimerAsynchronously(this, new TablistBroadcastHandler(this), 20L, this.tablistConfiguration.playerListUpdateInterval);
-        this.rankRecalculationTask = Bukkit.getScheduler().runTaskTimerAsynchronously(this, new RankRecalculationTask(pluginConfiguration, this.rankManager, this.userManager, this.guildManager), 20L, this.pluginConfiguration.rankingUpdateInterval);
+        this.topRecalculationTask = Bukkit.getScheduler().runTaskTimerAsynchronously(this, new TopRecalculationTask(this), 20L, this.pluginConfiguration.rankingUpdateInterval);
```

Comment:
Po co jest zmiana tej nazwy z rank na top, szczególnie jak to z domeny rank wychodzi